### DISCUSSION TITLE
[Feedback] Solar Flare Ticket Quests

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2371] [General]
Hello everyone, I would like to hear your comments about this event.

For me it was an excellent idea, but perhaps it was not well implemented. Merit should be the best way to distribute NFTs but its very massive, it takes a lot of time from players. This can even limit the arrival of new players..

I believe that with player feedback the developers will be able to think of something more attractive for next season.

**Comment by VZozul** [comment]
Yes, you're right, it takes a lot of time, nightly rises and it's bad.. it scares a little.. on the other hand, if you don't want to waste time, don't go through quests, and then buy SFT.. plant what people need for quests and sell. I thought about it, but didn't think of anything better. Quests are the perfect solution, the only way out is to do more long quests, long crops, a lot of gold mining, so that there is time to rest.

With the advent of land balance, it is very important to correctly distribute tickets for the next season. It would be ideal to make more SFTs that would require different areas. So that people do not have time to do everything, and so that no more than half of the total number of SFTs is mined. To do this, you need to run the Auction in parallel, which will be announced in advance, and will be held at the end of the season. That's when it will be interesting, then people will start to choose, besides this, do the same as with Iron Idol and Karkinos, what requires digging, not as difficult as Idol, but harder than Karkinos.

With the advent of the land balance, the RNG will leave, and a new problem will appear, the absence of the RNG. That is, everything will be distributed in hierarchies. Whales with boosts will get the best SFTs, the weakest ones will receive accordingly. And everything seems to be correct, as it should be, if you want to get more, grow! But how do you give them a chance? How to add a small RNG?
1. Every 10th quest is random. If you loop quests, after the last quest you need to do the previous one and down. Not at first, but in reverse order, but then after reaching 1 quest, then go from the end and down. 1-100, 99-1, 100-1. And it's even better to make the quests endless, yes, it takes time, but it's worth it.
2. Introduce Speed seeds (https://github.com/sunflower-land/sunflower-land/discussions/2357)
With speed seeds, there will be at least some RNG, and they should definitely not stack with existing boosts for carrots, parnship and cabbage. Give the weak a chance! Save the Speed seeds and save them for next season.
3. Add Tickets as rewards for Treasure Island, not a big chance, like with crops. This will burn resources and add RNG.
4. Add seaweed, sea cucumber, crab and starfish to the quests. Not just to give from Basket, but to get it by digging.
 
  There were a couple of very negative moments this season, it all started with additional tickets that were issued by chance, and okay, the supply was raised, but they still had the advantage of minting both the boy and the girl .. The next moment is that instead of 750 tickets for the boy, many were taken away only 700, and 50 remained, then it was fixed, but those who managed to get an advantage on other SFTs. And you can't do that! If there was already a bug, leave it for everyone until the end of the season, and then fix it, but everyone should have had an equal chance. If someone received more tickets by mistake, then give the rest the same number and raise the supply. It would be fair. And the way you get out of the situation, it's not fair!

**Comment by ghost** [comment]
I put forward a very interesting proposal to change the system of quests, you can read it: https://github.com/sunflower-land/sunflower-land/discussions/2326
